j5/components/piezo.py

```diff
@@ -31,12 +31,6 @@ class Note(float, Enum):
     B7 = 3951.1
     C8 = 4186.0
 
-    def __reverse__(self) -> Generator['Note', None, None]:
```

Comment:
:+1:

I don't think it was useful to have this function in our API anyway. On the offchance that someone needs a list of notes in reverse order, they can construct that sequence themselves quite easily.

Reply:
It actually works anyway. You can run `reversed(Note)` and it will iterate in reverse order. Hence why the method was removed but not the test.

Reply:
Ah, looks like this is just a side effect of enums being iterable:

```
Python 3.7.3 (default, Mar 25 2019, 20:59:09) 
[GCC 7.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from enum import Enum, auto
>>> class MyEnum(Enum):
...   A = auto()
...   B = auto()
...   C = auto()
... 
>>> list(MyEnum)
[<MyEnum.A: 1>, <MyEnum.B: 2>, <MyEnum.C: 3>]
>>> list(reversed(MyEnum))
[<MyEnum.C: 3>, <MyEnum.B: 2>, <MyEnum.A: 1>]
```